c/meterpreter/source/common/common_logging.c

```diff
@@ -0,0 +1,43 @@
+#include "common.h"
```

Comment:
This might make sense to extract as its own object build that extensions link in vs compile directly into each extension.

consider `c/meterpreter/source/logging/logging.c` or some similar name and using `common_logging.h` as a header file consumed by the extensions.

Reply:
I think what we have now is probably simple enough to keep as is but I think if we were to expand the capabilities of the logging feature in the future it could do with being it's own thing that extensions can link into

Reply:
We had discussed that having a `.c` source file in this `common` directory which maybe should be called `include` to more clearly represent that it is for shared header files, was not a desired precedent we want to set. I stand by that assertion in noting this should be refactored. Taking a moment to organize this seems prudent.